bin/dbconsole

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
rename file to mysql_console ?
or db_console ? 

naming consistency ?

Reply:
makes sense, will do